admin/tests-integration/sql.service.spec.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+/* global describe beforeEach beforeAll afterEach it expect jasmine spyOn fail */
+
+require('dotenv').config()
+const sql = require('../services/data-access/sql.service')
+const sqlPool = require('../services/data-access/sql.pool.service')
+const Connection = require('tedious').Connection
+const Request = require('tedious').Request
+
+const executeAdminRequest = async (adminConnection, sql) => {
```

Comment:
This might not need to be async.  As it returns a Promise it can still be await'ed on.

Reply:
good spot 👏 

golden rule - only mark as async if you need to await